scripts/scil_compute_local_tracking_2.py

```diff
@@ -0,0 +1,228 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Local streamline HARDI tractography using only scilpy methods -- no dipy (i.e
+no cython). The goal of this is to have a python-only version that can be
+modified more easily by our team when testing new algorithms and parameters,
+and that can be used as parent classes in sub-projects of our lab such as in
+dwi_ml.
+
+As in scil_compute_local_tracking:
+
+    The tracking direction is chosen in the aperture cone defined by the
+    previous tracking direction and the angular constraint.
+    - Algo 'det': the maxima of the spherical function (SF) the most closely
+    aligned to the previous direction.
+    - Algo 'prob': a direction drawn from the empirical distribution function
+    defined from the SF.
+    - Algo 'eudx' is not yet available!
```

Comment:
just remove that line. We will most likely never make eudx available, no?

Reply:
I wasn't sure. Ok.